l10n_th_withholding_tax_report/report/report_withholding_tax.py

```diff
@@ -78,4 +74,8 @@ def _compute_results(self):
             domain += [("cert_id.date", ">=", self.date_from)]
         if self.date_to:
             domain += [("cert_id.date", "<=", self.date_to)]
+        if self.company_id:
```

Comment:
Is self.company required?
you can add
`domain = [("cert_id.company_partner_id", "=", self.company_id.partner_id.id)]`
instead of condition.